oscduplicator/settings.py

```diff
@@ -42,11 +49,13 @@ def load_json(self) -> None:
                 port=element["port"],
                 enabled=element["enabled"],
             )
+        self.auto_start = data["auto_start"]
 
     def save_json(self):
         save_data = {
             "receive": {"port": self.receive_port},
             "transmit": list(map(asdict, self.transmit_port_settings)),
+            "auto_start": self.auto_start
```

Comment:
```suggestion
            "auto_start": self.auto_start,
```
カンマつけてみた（あとあと下に要素を追加するときにちょっとだけ楽）